src/workloads/execution/LastPoint.yml

```diff
@@ -0,0 +1,188 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of last point queries after the first milestone.
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        {create: &coll Collection6, timeseries: {timeField: "time", metaField: "tags"}}
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
```

Comment:
I thought so as well, however, when I was looking at `FillTimeseriesCollection.yml` from this PR ([link](https://github.com/mongodb/genny/pull/588)) I noticed that a Quiesce stage was used even after the creation stage for TimeSeries collections. So I added a Quiesce stage after the creation and loading data stages to stay on the safe side. Do you think it's a good idea to do away with this Quiesce stage?

Reply:
Yeah, they may have done that when testing $fill because there was a kind of loading latency for each query. Since we won't need to worry about that for this workload, we should only use the one quiesce stage - of course, we can revisit this decision if the benchmark results show differently.

Reply:
Makes sense, removed!